src/app/manager-dashboard/certifications/certifications-view.component.html

```diff
@@ -6,16 +6,26 @@
 </mat-toolbar>
 
 <div class="space-container">
-  <mat-toolbar class="primary-color font-size-1">
-    {{certification.name}}
+  <mat-toolbar class="primary-color font-size-1" >
+   {{certification.name}}
+   <span class="empty-spacer"></span>
+   <button mat-stroked-button (click)="routeToEdit(certification?._id)">
+    <mat-icon>edit</mat-icon>Edit
+   </button>
   </mat-toolbar>
   <div class="view-container view-full-height">
     <mat-tab-group>
       <mat-tab i18n-label label="Course Requirements">
-        <planet-courses *ngIf="certification.courseIds.length > 0" [displayedColumns] = "[ 'courseTitle', 'info', 'createdDate', 'rating' ]" [isForm]="true" [includeIds]="certification.courseIds"></planet-courses>
+        <planet-courses *ngIf="certification.courseIds.length > 0; else noCoursesMessage" [displayedColumns] = "[ 'courseTitle', 'info', 'createdDate', 'rating' ]" [isForm]="true" [includeIds]="certification.courseIds"></planet-courses>
```

Comment:
The `*ngIf` here isn't necessary here because there is no course table if there are no courses already.  You can just put a condition on the "No courses" message.

Reply:
@paulbert  on removing the *ngIf here, on the certifications/view page the Course Requirements displayed were not correct.